app/v2/index.js

```
@@ -1,6 +1,14 @@
 const v2 = module.exports = require('express').Router();
+const { membershipApiKey, membershipApiUrl } = require('../../config/config');
+
 const middleware = [
-	require('./middleware/caching')
+	require('cookie-parser')(),
+	require('./middleware/session')({
+		membershipApiUrl,
+		membershipApiKey
+	}),
+	require('./middleware/caching'),
+	require('./middleware/error')
```

This error middleware will serve as a catch-all handler for other potential errors in further PRs